samples/user-scope-web-application/nodejs/.env

```diff
@@ -0,0 +1,9 @@
+MicrosoftAppId = "<<MicrosoftAppId>>"
+MicrosoftAppPassword = "<<MicrosoftAppPassword>>"
+MicrosoftAppTenantId = "<<MicrosoftAppTenantId>>"
+AllowedHosts = "*"
+notificationUrl= "<<BaseUrl>>/api/notifications"
```

Comment:
To avoid duplicate entry, can you please use relative path for notificationUrl. It should be "/api/notifications". Internally you can add BaseUrl.

Reply:
fixed

Reply:
so you want me to add only this **"/api/notifications"** right ?